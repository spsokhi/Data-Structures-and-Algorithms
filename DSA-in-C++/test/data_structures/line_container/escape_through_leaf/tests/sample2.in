4
5 -10 5 7
-8 -80 -3 -10
2 1
2 4
1 3
