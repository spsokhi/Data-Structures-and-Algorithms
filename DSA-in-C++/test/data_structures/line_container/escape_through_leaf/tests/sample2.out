-300 100 0 0
